Framework/include/ScaleFactors.h

```diff
@@ -271,15 +272,25 @@ class ScaleFactors
                 }
                 
                 double eleTotSF         = eleTightSF*eleIsoSF*eleRecoSF*eleTrigSF; 
+                double eleNoTrigSF      = eleTightSF*eleIsoSF*eleRecoSF;
```

Comment:
I don't like that we are computing these multiplications twice. I think these two line should go like this:
const double eleNoTrigSF      = eleTightSF*eleIsoSF*eleRecoSF;
const double eleTotSF            = eleNoTrigSF*eleTrigSF;